Sources/MeiliSearch/Indexes.swift

```diff
@@ -26,7 +26,7 @@ public struct Indexes {
   private let documents: Documents
 
   // Search methods
-  private let search: Search
+  fileprivate let search: Search
```

Comment:
This change is actually not required - `private` will work just as fine.

Reply:
Let me double check this, I thought I had a compiler error when I tested (due to use of an extension) but will double check and revert if possible.

In respect of "I have not actually tested it though" - I'm using this in a production application and is working fine for me 🙂 Though I'll keep an eye on it!

Reply:
Yeah that's fair, agreed. (If there is value in hiding these properties internally we could still use protocols for that - though not sure if it is necessary.)

Only final point that I'd make then is that I would not explicitly state `internal` - given that this is the default anyway and not used on other internal properties.

Reply:
Fair point, that's a personal preference but deviates from standard practice for the project. I'll amend in a follow up PR